rust/rsc/src/rsc/main.rs

```diff
@@ -482,8 +482,8 @@ mod tests {
                     .header("Authorization", api_key)
                     .body(Body::from(
                         serde_json::to_vec(&json!({
-                            "cmd": "blarg",
-                            "env":"PATH=/usr/bin",
+                            "cmd": [98, 108, 97, 114, 103], // blarg
```

Comment:
I think we should add/use some helper here so test writers don't have to manually to str.to_bytes.

Reply:
that was way easier than I had thought it would be